specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2016-05-01/ScaleUnit.json

```diff
@@ -288,6 +331,86 @@
           "type": "boolean"
         }
       }
+    },
+    "NetworkDefinitionParameter": {
+      "description": "A definition of the network received from a new cluster operation.",
+      "type": "object",
+      "properties": {
+        "subnet": {
+          "description": "The subnet IP mask in the example format 10.0.0.0/25.",
+          "type": "array",
+          "items": {
+             "type": "string"
+          }
+        },
+        "vlanId": {
+          "description": "The Vlan ID of the subnet.",
+          "type": "array",
+          "items": {
+             "type": "string"
+          }
+        }
+      }
+    },
+    "DeploymentJsonPhysicalNodeParameters": {
+      "description": "Description of a bare metal node used for scale unit scale out operations.",
+      "type": "object",
+      "properties": {
+        "bmcIpAddress": {
+          "x-ms-client-name": "BMCIPAddress",
+          "description": "BMC address of the physical machine.",
+          "type": "string"
+        },
+        "name": {
+          "description": "Computer name of the physical machine.",
+          "type": "string"
+        }
+      }
+    },
+    "CreateFromJsonScaleUnitParametersList": {
+      "description": "A list of input data that allows for creating the new scale unit.",
+      "type": "object",
+      "properties": {
+        "clusterName": {
```

Comment:
'clusterName' got removed from the sample bc is the same as the scaleUnit name...should be removed from here?

Reply:
No and I think it is wrong for me to remove it from the sample, I will put it back. I've looked over the code again to try to figure it out. (I did not write this code, it was Lincoln). The customer can call in two different ways, either supplying the name or not supplying the name.

Reply:
To supply the name it is a POST operation on ..\scaleUnit\<name>  and the name is in the body data. The code only uses the name in the body, the name on the URL is not used.
To create a cluster where a name is chosen by default, user does a POST on ..\scaleUnit. In that case the clusterName in the body should be null, although we don't check for that.

Reply:
sounds good

Reply:
Never mind... digging through the code more the clusterName parameter in the body does not get used... originally it looked like it did. So not sure why it was defined. I've removed it from the Swagger and example.